dts/riscv/nordic/nrf54h20_cpuflpr.dtsi

```diff
@@ -27,6 +27,11 @@ cpusys_vevif: &cpusys_vevif_tx {};
 		compatible = "simple-bus";
 		interrupt-parent = <&cpuflpr_clic>;
 		ranges;
+
+		stmesp: memory@ae000000 {
+			compatible = "arm,stmesp";
+			reg = <0xad000000 0x1000000>;
+		};
```

Comment:
is this exclusive to cpuflpr? can't it be part of the global file?

Reply:
it's a register set dedicated for flpr. Used same approach as in ppr case: https://github.com/zephyrproject-rtos/zephyr/blob/57a8a7c3500a6444448276103e890e5efbcd3a90/dts/riscv/nordic/nrf54h20_cpuppr.dtsi#L31

Reply:
ok, btw, 

```
stmesp: memory@ae000000` but `reg = <0xad000000 0x1000000>;
                ^~~                     ^~~
```

Reply:
fixed.